### PR TITLE
#170764519 Setup SASS processing on the client

### PR DESCRIPTION
**Pull request by Skemc**
#### What does this PR do?
Setup SASS processing on the client
#### Description of Task to be completed?
- add sass loader
- for using both sass and css
#### How should this be manually tested?

- You have to clone the repo `git clone https://github.com/Stackup-Rwanda/fan7-bn-frontend.git`
- Then After the repo cloned `cd project-folder` and run `npm i` to install all dependencies
- Go to my branch `git checkout ch-setup-sass-170764519`
- Run app by `npm run dev`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#170764519](https://www.pivotaltracker.com/story/show/170764519)
#### Screenshots (if appropriate)
N/A
#### Questions 
N/A

### CODE REVIEW

**Review by Elvis-rugamba — APPROVED** [review]
Resolve conflicts

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-node_modules
+node_modules/
 dist
-package-lock.json
 coverage
 .env
-.vscode
\ No newline at end of file
+.vscode
+<<<<<<< HEAD
```
Comment (shaazk) [inline .gitignore]: Remove these from your .gitignore

File: .gitignore [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+package-lock.json
+=======
```
Comment (shaazk) [inline .gitignore]: Remove these too

**Review by shaazk — APPROVED** [review]
LGTM

**Review by ajimae — APPROVED** [review]
LGTM!

**Pull request merged**